tests/python_client/testcases/test_utility.py

```diff
@@ -601,9 +601,9 @@ def test_rename_collection_not_existed_collection(self):
         new_collection_name = cf.gen_unique_str(prefix)
         self.utility_wrap.rename_collection(old_collection_name, new_collection_name,
                                             check_task=CheckTasks.err_res,
-                                            check_items={"err_code": 1,
-                                                         "err_msg": "can't find collection: {}".format(
-                                                             collection_w.name)})
+                                            check_items={"err_code": 4,
+                                                         "err_msg": "collection 1:test_collection_non_exist: "
```

Comment:
what does this mean?

Reply:
> what does this mean?

collection 1 means the first param of rename_collection, 
test_collection_non_exist is old_collection_name.